tdrs-frontend/src/components/SubmissionHistory/ReprocessedModal.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import Modal from '../Modal'
+
+const Message = ({ date }) => {
+  return (
+    <>
+      We've reprocessed your submission with updated validation criteria, based
+      on system improvements, to improve accuracy of error reports. No changes
+      have been made to your original data submission.
+      <br />
+      <br />
+      Data was reprocessed on: {date}
+    </>
+  )
+}
+
+const ReprocessedModal = ({ date }) => {
+  const [modalVisible, setModalVisible] = useState(false)
```

Comment:
related to my other comment but wanted to call this out specifically - it's generally advised to keep state as high up in the component chain as makes sense. this goes along with having a single modal controlled by the root page. 
- https://legacy.reactjs.org/docs/lifting-state-up.html